pulsar-client-cpp/include/pulsar/c/consumer_configuration.h

```diff
@@ -281,6 +281,12 @@ PULSAR_PUBLIC void pulsar_consumer_configuration_set_priority_level(
 PULSAR_PUBLIC int pulsar_consumer_configuration_get_priority_level(
     pulsar_consumer_configuration_t *consumer_configuration);
 
+PULSAR_PUBLIC int pulsar_consumer_is_replicate_subscription_state_enabled(
+    pulsar_consumer_configuration_t *consumer_configuration);
```

Comment:
Can you add explanations (docs) for your code change?

Example:
![image](https://user-images.githubusercontent.com/50226895/142128215-c1602334-4719-4c39-b6af-a5581c2f1822.png)

Reply:
BTW, there are some issues in C, welcome to make contributes to [this issue](https://github.com/apache/pulsar/issues/12843), thanks,